src/lib/dnssd/minimal_mdns/AddressPolicy_LibNlImpl.h

```diff
@@ -20,19 +20,7 @@
 
 namespace mdns {
 namespace Minimal {
-namespace LibNl {
 
-class LibNl_AddressPolicy : public mdns::Minimal::AddressPolicy
-{
-public:
-    chip::Platform::UniquePtr<ListenIterator> GetListenEndpoints() override;
-
-    chip::Platform::UniquePtr<IpAddressIterator> GetIpAddressesForEndpoint(chip::Inet::InterfaceId interfaceId,
-                                                                           chip::Inet::IPAddressType type) override;
-};
-
-void SetAddressPolicy();
-
-} // namespace LibNl
+AddressPolicy * GetDefaultAddressPolicy();
```

Comment:
Same here: this header should probably not exist at all. Declaration of this should be in `AddressPolicy.h`